backend/substrapp/utils.py

```diff
@@ -313,3 +313,7 @@ def timed(*args, **kw):
         return result
 
     return timed
+
+
+def new_uuid():
```

Comment:
I'm not sure this is necessary, If I need to create a new UUID my first thought would be to use the uuid.uuid4() library, not the utils function.
I think It can be removed as it changes nothing to the library behavior

Reply:
That makes sense. I was unsure whether or not to keep this function :)

Reply:
Fixed by https://github.com/SubstraFoundation/substra-backend/pull/342/commits/f23943fe9fdaefa65da74a40d845297a43bcd358